x-pack/plugin/sql/qa/src/main/resources/docs/docs.csv-spec

```diff
@@ -607,7 +607,7 @@ M              |57
 groupByAndAggExpression
 // tag::groupByAndAggExpression
 schema::g:s|salary:i
-SELECT gender AS g, ROUND((MIN(salary) / 100)) AS salary FROM emp GROUP BY gender;
+SELECT gender AS g, ROUND( (MIN(salary) / 100) ) AS salary FROM emp GROUP BY gender;
```

Comment:
I know this is in the file called `docs.csv-spec` but I'm worried that folks will remove the spacing without realizing the change that it makes in the docs. I wonder if there is a better way to change this. Let me look around some!